### PR TITLE
Upgrade to Julia 1.3+, use JLL package for ImageMagick

### PR DESCRIPTION
**Pull request by staticfloat**
Slightly simplify a few things as we can now directly specify what version of a dependency we support.

This is, naturally, 1.3+ only, I'm using it to try and make the [GtkArtifactTest](https://github.com/staticfloat/GtkArtifactTest) repo 100% JLL package based.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -3,22 +3,21 @@ uuid = "6218d12a-5da1-5696-b52f-db25d2ecc6d1"
...
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
-julia = "1"
-BinaryProvider = "≥ 0.3.0"
 ColorTypes = "≥ 0.2.7"
 FixedPointNumbers = "≥ 0.3.0"
 ImageCore = ">= 0.7.0"
+julia = "≥ 1.3"
```
Comment (timholy) [inline Project.toml]:
```suggestion
julia = "^1.3"
```
General consensus seems to be that you don't want to promise compatibility with Julia 2.x.

**Review by timholy — COMMENTED** [review]
Love it!

File: Project.toml (hunk 1 above)
Comment (staticfloat) [reply Project.toml]: Doesn't this mean that it won't be installed on Julia 1.4?  Maybe instead we should use `julia = "^1"`?

Comment (timholy) [reply Project.toml]: Nope, `^1.3` means any `1.x >= 1.3`. See https://julialang.github.io/Pkg.jl/v1/compatibility/#Caret-specifiers-1

Comment (staticfloat) [reply Project.toml]: You learn something new every day!

**Comment by timholy** [comment]
We're failing on nightly on a couple of platforms, but since it works on 1.3 I say let's merge this!

I might wait a day or so and see if we can fix nightly before registering a new version. Any reason not to call it 1.0?

**Pull request merged**